src/Component/Provider/DefaultInjectingEntityRepositoryDecorator.php

```diff
@@ -63,9 +63,9 @@ public function create(array $data, Context $context): EntityWrittenContainerEve
         return $this->decorated->create($this->injectDefaults($data, $context), $context);
     }
 
-    public function delete(array $data, Context $context): EntityWrittenContainerEvent
+    public function delete(array $ids, Context $context): EntityWrittenContainerEvent
```

Comment:
It makes sense to adopt the argument names of the interface. I can imagine, this improves support for [named arguments](https://www.php.net/manual/en/functions.arguments.php#functions.named-arguments).

I'll leave this link here for reference: https://github.com/shopware/platform/blob/v6.4.11.1/src/Core/Framework/DataAbstractionLayer/EntityRepositoryInterface.php#L31